app/util/ContentUpgrade.scala

```diff
@@ -1,14 +1,15 @@
 package util
 
 import com.gu.contentapi.client.model.v1.Content
-import com.gu.contentapi.json.JsonParser
-import com.gu.contentapi.json.JsonParser._
 import com.gu.facia.api.utils.{CardStyle, ResolvedMetaData}
 import com.gu.facia.client.models.TrailMetaData
 import org.json4s.JValue
 import org.json4s.JsonAST._
 import org.json4s.JsonDSL._
-import org.json4s.native.JsonMethods
+import org.json4s.jackson.JsonMethods
+import io.circe._
+import io.circe.parser._
```

Comment:
Is it possible to be more explicit about what you are importing from `circe`, such as `parse`?